pkg/workflows/wasm/host/wasm_test.go

```diff
@@ -75,6 +79,9 @@ func Test_GetWorkflowSpec(t *testing.T) {
 	spec, err := GetWorkflowSpec(
 		&ModuleConfig{
 			Logger: logger.Test(t),
+			Fetch: func(req wasm.TargetRequestPayload) (wasm.TargetResponsePayload, error) {
```

Comment:
@agparadiso GetWorkflowSpec shouldn't need to provide a Fetch implementation: can we modify the passed in config with a noop'd implementation of Fetch instead? The Fetch API shouldn't be available when making a GetWorkflowSpec call